src/dune_rules/dune_file.ml

```diff
@@ -128,7 +128,7 @@ module Buildable = struct
     ; empty_module_interface_if_absent : bool
     ; libraries : Lib_dep.t list
     ; foreign_archives : (Loc.t * Foreign.Archive.t) list
-    ; foreign_objects : Foreign.Objects.t
+    ; extra_objects : Foreign.Objects.t
```

Comment:
This naming mismatch suggests that there is scope for further refactoring. (Not necessarily in this PR.)

Reply:
Good point! Think it can just go in a new file (objects.ml)?